### PR TITLE
Upgrade from v10 to v11

### PR DESCRIPTION
**Pull request by megian**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

**Comment by megian** [comment]
Requires #176, #175, #158, #180.

File: docs/modules/ROOT/pages/how-tos/upgrade-10.x-to-11.x.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+= Upgrade from v10 to v11
+
+This guide describes the steps to perform an upgrade of the component from version v9 to v10.
+
+== Changes
+
+* The component requires at least a Kubernetes version of v1.21.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/upgrade-10.x-to-11.x.adoc:7]:
```suggestion
* The component requires Kubernetes  v1.21 or newer.
```

File: docs/modules/ROOT/pages/how-tos/upgrade-10.x-to-11.x.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* K8up v1 support dropped.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/upgrade-10.x-to-11.x.adoc:8]:
```suggestion
* K8up v1 support has been removed.
```

**Review by simu — APPROVED** [review]
LGTM

**Pull request merged**